spring/boot2-actuator-autoconfigure/src/main/java/com/linecorp/armeria/spring/actuate/WebOperationServiceUtil.java

```diff
@@ -0,0 +1,16 @@
+package com.linecorp.armeria.spring.actuate;
+
+import org.springframework.boot.actuate.endpoint.OperationArgumentResolver;
+import org.springframework.boot.actuate.endpoint.ProducibleOperationArgumentResolver;
+
+import com.linecorp.armeria.common.HttpHeaderNames;
+import com.linecorp.armeria.common.HttpHeaders;
+
+final class WebOperationServiceUtil {
+
+    static OperationArgumentResolver acceptHeadersResolver(HttpHeaders headers) {
```

Comment:
How about putting this method in `WebOperationService` because this is only used there?

Reply:
I intentionally extracted it to avoid 
`ClassNotFoundException`. It is possible to make this a nested class.